source/StartConditionsPanel.h

```diff
@@ -0,0 +1,68 @@
+/* StartConditionsPanel.h
+Copyright (c) 2020 by Michael Zahniser
+
+Endless Sky is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free Software
+Foundation, either version 3 of the License, or (at your option) any later version.
+
+Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+*/
+
+#ifndef START_CONDITIONS_PANEL_H_
+#define START_CONDITIONS_PANEL_H_
+
+#include "Panel.h"
+
+#include "ClickZone.h"
+#include "Color.h"
+#include "LoadPanel.h"
+#include "PlayerInfo.h"
+#include "text/WrappedText.h"
+
+class StartConditionsPanel : public Panel {
+public:
+	StartConditionsPanel(PlayerInfo &player, UI &gamePanels, LoadPanel *LoadPanel);
+	void OnCallback(int);
+	virtual void Draw() override;
+	
+	
+protected:
+	// Only override the ones you need; the default action is to return false.
+	virtual bool KeyDown(SDL_Keycode key, Uint16 mod, const Command &command, bool isNewPress) override;
+	virtual bool Click(int x, int y, int clicks) override;
+	virtual bool Hover(int x, int y) override;
+	virtual bool Drag(double dx, double dy) override;
+	virtual bool Scroll(double dx, double dy) override;
+	
+	
+private:
+	PlayerInfo &player;
+	UI &gamePanels;
+	
+	WrappedText descriptionWrappedText;
+	
+	// Workaround around the absence of std::optional<>
+	StartConditions chosenStart;
```

Comment:
Should I also turn StartConditions into a pointer in other classes such as PlayerInfo?

Reply:
Yeah, a new player should copy the start summary, and loading a save should read that in. It should be separate from those in GameData since the player might remove the source plugin, or it may get changed after the player begins their game.

This does mean we won't be able to update existing player's start data after they begin (e.g. setting some conditions based on future changes) but also means players don't lose start dates, etc. just because their "upstream" start condition was modified